### ISSUE TITLE
Double tap seeking on Mobile like Youtube

### ISSUE THREAD

**Issue by Abdulmanan0315** [sampotts/plyr#1363]
I have a feature request that most people will love and it is **tapping on the half of the player screen on Right seeks Forward 10 seconds and tapping on the half of the player screen on left seek Backward 10 seconds** , just like the Youtube app .
This will help improve video seeking on Mobile devices.
We 'll love to hear more.
Thanks for the great plugin.

**Comment by mafmartins** [comment]
I also agree this would be a great feature, the major mobile HTML players are using this feature.

**Comment by aykuxt** [comment]
Is there any progress about this feature request?

**Comment by mabuckman** [comment]
+1

**Comment by matpo** [comment]
Any news about this ? Or do you have advises to implement it ourselves ?
